### ISSUE TITLE
Update gazebo5 import to bring in sdformat-sdf

### ISSUE THREAD

**Issue by tfoote** [ros-infrastructure/ros_bootstrap_dependencies#3]
```
--- a/ubuntu/config/gazebo5.upstream.yaml
+++ b/ubuntu/config/gazebo5.upstream.yaml
@@ -3,4 +3,5 @@ method: http://packages.osrfoundation.org/gazebo/ubuntu
 suites: [trusty, utopic]
 component: main
 architectures: [i386, amd64, source]
-filter_formula: Package (% gazebo5-dbg ) | Package (% gazebo5 ) | Package (% libsdformat2) | Package (% libsdformat2-dbg) | Package (% libsdf
+filter_formula: Package (% gazebo5-dbg ) | Package (% gazebo5 ) | Package (% libsdformat2) | Package (% libsdformat2-dbg) | Package (% libsdf
+
```

The question is do we want the new versions of the other debs?

```
update 'gazebo5' '5.0.1-1~trusty' '5.1.0-1~trusty' 'gazebo5_import'
update 'gazebo5-dbg' '5.0.1-1~trusty' '5.1.0-1~trusty' 'gazebo5_import'
update 'libsdformat2' '2.3.0-1~trusty' '2.3.2-1~trusty' 'gazebo5_import'
update 'libsdformat2-dbg' '2.3.0-1~trusty' '2.3.2-1~trusty' 'gazebo5_import'
update 'libsdformat2-dev' '2.3.0-1~trusty' '2.3.2-1~trusty' 'gazebo5_import'
```

I manually held them back for now.

@jrivero @scpeters

**Comment by j-rivero** [comment]
The main point is if we want to backport the same version that is officially into Ubuntu/Vivid (5.0.1+dfsg-1~exp2) to Utopic and Trusty. I can build it the OSRF Jenkins but do not export it to packages.osrfoundation.org and you can probably grab them directly from the jenkins job. 

I think that this was proposed by @scpeters and agree by most of us. So green light?

**Comment by scpeters** [comment]
:+1:

**Comment by j-rivero** [comment]
Backported versions:
- [Utopic distribution](http://build.osrfoundation.org/job/gazebo5_vivid-debian-debbuilder/14/)
- [Trusty distribution](http://build.osrfoundation.org/job/gazebo5_vivid-debian-debbuilder/14/)

The source package and the rest of the files needed for a source build are also available.

**Comment by scpeters** [comment]
[utopic link here](http://build.osrfoundation.org/job/gazebo5_vivid-debian-debbuilder/16/)

`gazebo5` was split into multiple packages, so we need more packages added:

```
gazebo5-common
gazebo5-doc
gazebo5-plugin-base
gazebo5
libgazebo-dev
libgazebo5-dbg
libgazebo5-dev
libgazebo5
```

@j-rivero are all of these needed? (namely `libgazebo-dev` and `libgazebo5-dev`)

**Comment by j-rivero** [comment]
optionals: gazebo5-doc and libgazebo5-dbg, all the rest are needed for ROS.

Thanks for fixing the link to utopic.

**Comment by scpeters** [comment]
Currently, I only see `gazebo5` and `gazebo5-dbg` in utopic, and they are unable to install without the other packages.

**Comment by tfoote** [comment]
So these packages need libsimbody3.5 I'll add that to the import from the repos. The import also brought in libsdformat 2.3.2

**Comment by j-rivero** [comment]
That is right, needed dependencies of gazebo5, not present in Utopic or Trusty.

**Comment by scpeters** [comment]
Ok, let me know when they are imported, and I will test.

**Comment by tfoote** [comment]
@scpeters Thanks I've deployed it to our test farm here: http://54.183.65.232/ubuntu/testing/

**Comment by tfoote** [comment]
That's equivalent to shadow-fixed

**Comment by scpeters** [comment]
What's the key for that repo?

**Comment by scpeters** [comment]
nevermind I found it

**Comment by scpeters** [comment]
@j-rivero those links are only for 64-bit trusty and utopic. Can you build 32-bit debs as well?

**Comment by scpeters** [comment]
We should also remove the vivid debs of gazebo5:

http://54.183.65.232/ubuntu/testing/pool/main/g/gazebo5/

**Comment by j-rivero** [comment]
Sure:
- [i386 utopic](http://build.osrfoundation.org/job/gazebo5_vivid-debian-debbuilder/20/)
- [i386 trusty](http://build.osrfoundation.org/job/gazebo5_vivid-debian-debbuilder/19/)

**Comment by tfoote** [comment]
I've pulled the 386 builds in and propogated to the test farm. @scpeters please test. I'll clean up the vivid debs separately.

**Comment by scpeters** [comment]
Looks good so far. I'm installing trying an i386 build on utopic as well. There some conflicts with some low-level packages that prevents side-by-side installation, so I'm removing most of the packages before reinstalling

**Comment by scpeters** [comment]
I just released gazebo_ros_pkgs for jade with @wjwwood 's help, so we'll see if the build farm has any problems.

**Comment by scpeters** [comment]
The gazebo5 trusty debs were depending on libogre-1.9-dev, though [we had decided](https://github.com/ros-infrastructure/rep/pull/89#issuecomment-69232117) to use ogre 1.8 on trusty. We rebuilt the debs to use ogre 1.8 on trusty:
- http://build.osrfoundation.org/job/gazebo5_vivid-debian-debbuilder/23/
- http://build.osrfoundation.org/job/gazebo5_vivid-debian-debbuilder/24/

@tfoote can you import these?

**Comment by tfoote** [comment]
importing done, it should be in all the build environments, the next shadow-fixed sweep it will go out and the next public release they will got into the public repos
